src/Microsoft.DotNet.XHarness.CLI/Commands/XHarnessHelpCommand.cs

```diff
@@ -39,7 +40,15 @@ public override int Invoke(IEnumerable<string> arguments)
                     PrintCommandHelp(new AndroidCommandSet(), subCommand);
                     break;
                 case "ios":
-                    PrintCommandHelp(new iOSCommandSet(), subCommand);
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                    {
+                        PrintCommandHelp(new iOSCommandSet(), subCommand);
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Command 'ios' could be run on OSX only.");
```

Comment:
nit:

```suggestion
                        Console.WriteLine($"Command '{command}' could be run on OSX only.");
```

because we will be changing it soon so that we don't forget it here..